### ISSUE TITLE
mu4e-views-next-previous-message-behaviour not working

### ISSUE THREAD

**Issue by Aquan1412** [lordpretzel/mu4e-views#19]
The setting `mu4e-views-next-previous-message-behaviour` doesn't work for me, the focus of mu4e always switches to the message view after changing the email (by pressing n or p). However, I want it to stay focused on the header view. I tried both `stick-to-current-window` and `always-switch-to-headers`, but the behaviour didn't change.

I'm currently using Emacs 27.2. If you need additional information, please let me know.

**Comment by lordpretzel** [comment]
Hi, are you happen to use linux? If yes, it seems that the xwidget always captures focus when browsing a new http doc (email). If that is the case, then currently, I don't think there is anything I could do about that except maybe for a dirty hack that runs a timer and then switches to the headers view.

**Comment by Aquan1412** [comment]
Hi, yes, I forgot to mention that I'm using Manjaro Linux, so that would explain the issue. Too bad, but if it's a problem with xwidget then it's probably best to wait for an upstream fix. For now, I'll simply use “y” to change focus manually if I need it. I fear, the workaround with the timer might be confusing, if the focus changes from the headers to the mail and back to the header.

**Comment by lordpretzel** [comment]
yes, that would be a mess, especially since there is good choice for how long the timer would run so sometimes it would switch after another user interaction and sometimes too early when loading the html in xwidgets takes longer.
